test/tyche/YieldDirector.test.ts

```diff
@@ -112,13 +109,15 @@ describe.only("YieldDirector", async () => {
         ohm = await ohmFactory.deploy(auth.address);
         sOhm = await sOhmFactory.deploy();
         gOhm = await gOhmFactory.deploy(sOhm.address, sOhm.address); // Call migrate immediately
+        const blockNumBefore = await ethers.provider.getBlockNumber();
+        const blockBefore = await ethers.provider.getBlock(blockNumBefore);
         staking = await stakingFactory.deploy(
             ohm.address,
             sOhm.address,
             gOhm.address,
-            "10",
+            "28800", // 1 epoch = 8 hours
```

Comment:
interesting, why did the 10 seconds not work?

Reply:
yea 10 seconds is too short by time deposit happens its ready to rebase and does that thing that stuffs up the distribute.